modules/hcp-virt-reqs.adoc

```diff
@@ -47,6 +47,8 @@ $ oc patch storageclass ocs-storagecluster-ceph-rbd -p '{"metadata": {"annotatio
 $ oc get managedclusters local-cluster
 ----
 
+* On the {product-title} cluster that hosts the {VirtProductName} virtual machines, you must use a `ReadWriteMany` (RWX) storage class so that live migration can be enabled.
```

Comment:
Can we add another requisite here that the management cluster can only be on-prem baremetal?